pipelines/chat/delete_message.fp

```diff
@@ -18,6 +18,18 @@ pipeline "delete_message" {
     description = "Timestamp of the message to be deleted."
   }
 
+  step "pipeline" "list_channels" {
```

Comment:
Can we update the param channel as channel_id and use it in the API directly? As param `channel` is not clear on name or id.